sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -854,10 +860,17 @@ __SYCL_EXPORT bool is_source_kernel_bundle_supported(backend BE,
 // syclex::create_kernel_bundle_from_source
 /////////////////////////
 
+kernel_bundle<bundle_state::ext_oneapi_source>
+create_kernel_bundle_from_source(const context &SyclContext,
+                                 source_language Language,
+                                 sycl::detail::string_view Source);
 __SYCL_EXPORT kernel_bundle<bundle_state::ext_oneapi_source>
 create_kernel_bundle_from_source(const context &SyclContext,
                                  source_language Language,
-                                 const std::string &Source);
+                                 const std::string &Source) {
+  return create_kernel_bundle_from_source(SyclContext, Language,
+                                          sycl::detail::string_view(Source));
+}
```

Comment:
Since it now has its definition in the headers, different includes of the header may think they have a new definition of it. Adding `inline` before the return type of the function should address it.

Additionally, `__SYCL_EXPORT` should be moved to the new variants.

Reply:
Thanks. Adding inline solved the issue.

> Additionally, `__SYCL_EXPORT` should be moved to the new variants.
The new function is not in the spec, so it should not be exported.